src/main/resources/application.yml

```
@@ -146,6 +146,10 @@ rest:
 iuv:
   informationSystemId: "00"
 
+installment-paid-view:
+  max-months-interval: "\${INSTALLMENT_PAID_VIEW_MAX_MONTHS_INTERVAL: 6}"
+  max-total-elements: "\${INSTALLMENT_PAID_VIEW_MAX_TOTAL_ELEMENTS: 100000}"
```

let's give a context.
let's introduce a default config for all common export limits (are also the month range a common limit?)
```suggestion
data-export:
  default:
     max-total-elements: "\${DATA_EXPORT_MAX_TOTAL_ELEMENTS: 100000}"
  installment-paid-view:
    max-months-interval: "\${INSTALLMENT_PAID_VIEW_MAX_MONTHS_INTERVAL: 6}"
    max-total-elements: "\${INSTALLMENT_PAID_VIEW_MAX_TOTAL_ELEMENTS:\${data-export.default.max-total-elements}}"
```